raml-parser-2/src/test/resources/org/raml/v2/parser/traits-and-resource-types/parameters-missing/output.txt

```diff
@@ -158,7 +158,7 @@ RamlDocumentNode (Start: 11 , End: 441, On: input.raml, Source: SYObjectNode)
                     KeyValueNodeImpl (Start: -1 , End: -1)
                         StringNodeImpl: "value" (Start: -1 , End: -1)
                         OverlayableStringNode: "/broken" (Start: -1 , End: -1, Source: SYStringNode)
-            KeyValueNodeImpl (Start: 76 , End: 131, On: input.raml, Source: KeyValueNodeImpl)
+            ErrorNode: "Unexpected key 'ErrorNode [Cannot resolve parameter method]'. Options are : "description" or "displayName" or "is" or "securedBy" or "type" or "uriParameters" or "usage" or Any of :Any of :"get","patch","put","post","delete","options","head",Any of :"get?","patch?","put?","post?","delete?","options?","head?"" (Start: 76 , End: 131, On: input.raml, Source: KeyValueNodeImpl)
```

Comment:
The error message is misleading, we can merge this pr but create another issue to improve/fix this message